src/stdfblib/events/E_TimeOut_fbt.h

```diff
@@ -43,7 +43,7 @@ class FORTE_E_TimeOut final : public CEventSourceFB {
     FORTE_E_TimeOut(CStringDictionary::TStringId paInstanceNameId, forte::core::CFBContainer &paContainer);
     bool initialize() override;
 
-    FORTE_ATimeOut var_TimeOutSocket;
+    FORTE_ATimeOut_Socket var_TimeOutSocket;
```

Comment:
I would suggest changing this to:
```suggestion
    forte::core::CInternalFB<FORTE_ATimeOut_Socket> fb_TimeOutSocket;
```

That way, we can avoid explicitly delegating to adapters in `initailize()` and also do not need special handling in `CFunctionBlock::getFB`.

Reply:
done. But I still kept the getAdapterUnchecked because it is used a lot in different places.